scripts/setup-benchmarks/install_ycsb.sh

```diff
@@ -0,0 +1,13 @@
+# Update and install required packages
+apt update
+
+# Download YCSB
+curl -O --location https://github.com/brianfrankcooper/YCSB/releases/download/0.17.0/ycsb-0.17.0.tar.gz
+tar xfvz ycsb-0.17.0.tar.gz
+cd ycsb-0.17.0
+
+# Copy contents of ycsb_runner.py to bin/ycsb
+sudo cp ../scripts/setup-benchmarks/ycsb_runner.py bin/ycsb
```

Comment:
why does this need sudo?

Reply:
bin/ycsb needs elevated privileges to modify

Reply:
that is not true, I just confirmed by downloading and deleting that file

I suspect that you originally downloaded the tar file or unzipped it as root (especially since you had `apt update` that requires root), and in that case you would need root to modify the file

please try running this all without root and things should just work